### PR TITLE
Quickstart netstats

### PR DESCRIPTION
**Pull request by schmir-at-bb**
This implements https://github.com/trustlines-network/project/issues/462

### CODE REVIEW

File: quickstart.sh [hunk 1]
```diff
@@ -353,6 +349,69 @@ The Parity client has started and is running in background!
...
+# HIDE_VALIDATOR_STATUS=false
+EOF
+}
+
+function startNetstats() {
+  stopAndRemoveContainer ${DOCKER_CONTAINER_NETSTATS}
+  printmsg <<EOF
+Starting netstats container...
+EOF
+  ${PERMISSION_PREFIX} docker run --network ${DOCKER_NETWORK} --name ${DOCKER_CONTAINER_NETSTATS} -d --restart=always --env-file "${NETSTATS_ENV_FILE}" -e RPC_HOST=${DOCKER_CONTAINER_PARITY} ${DOCKER_IMAGE_NETSTATS}
```
Comment (weilbith) [inline quickstart.sh]:
```suggestion
  ${PERMISSION_PREFIX} docker run \
    --detach \
    --restart=always \
    --network ${DOCKER_NETWORK} \
    --name ${DOCKER_CONTAINER_NETSTATS} \
    --env-file "${NETSTATS_ENV_FILE}" \
    --env RPC_HOST=${DOCKER_CONTAINER_PARITY} \
    ${DOCKER_IMAGE_NETSTATS}
```

Comment (weilbith) [reply quickstart.sh]: Writing out parameter names and split way too long command to multiple lines for readability.

File: quickstart.sh [hunk 2]
```diff
@@ -373,8 +432,16 @@ EOF
     setupAccountInteractive
   fi
 
+  if  [[ ! -e ${NETSTATS_ENV_FILE} ]]; then
```
Comment (weilbith) [inline quickstart.sh]:
```suggestion
  if  [[ ! -f ${NETSTATS_ENV_FILE} ]]; then
```

Comment (weilbith) [reply quickstart.sh]: I think this is more precise and also what we actually need, especially 6 lines below.

File: quickstart.sh [hunk 3]
```diff
@@ -353,6 +349,69 @@ The Parity client has started and is running in background!
...
+  if [[ "$(askYesOrNo 'Did you already receive credentials and want to setup the netstats client?')" = "no" ]]; then
+    return 0
+  fi
+  while true; do
+    local username
+    local password
+    read -r -p "Username: " username
+    read -s -r -p "Password: " password
+    if checkCredentials https://laikanetstats.trustlines.foundation/check/ "${username}:${password}" ; then
+      printmsg <<EOF
```
[the remark this answers is not in the document]
Comment (schmir-at-bb) [reply quickstart.sh]: I've moved it out of the loop.

File: quickstart.sh (hunk 2 above)
Comment (schmir-at-bb) [reply quickstart.sh]: I've changed all of the -e tests...

File: quickstart.sh [hunk 4]
```diff
@@ -353,6 +349,69 @@ The Parity client has started and is running in background!
 EOF
 }
 
+function checkCredentials() {
+  local url
```
[the remark this answers is not in the document]
Comment (schmir-at-bb) [reply quickstart.sh]: I think I slightly prefer multiple lines.

**Comment by schmir-at-bb** [comment]
I've reformatted the file with shfmt, with '-i2' because I don't like the indentation with tabs, which it uses by default and we already indent with 2 spaces.

It's not integrated into circleci however at the moment.

**Pull request merged**